packages/pages-shared/vitest.config.mts

```diff
@@ -1,12 +1,15 @@
-import { defineProject, mergeConfig } from "vitest/config";
-import configShared from "../../vitest.shared";
+import { defineWorkersProject } from "@cloudflare/vitest-pool-workers/config";
 
-export default mergeConfig(
-	configShared,
-	defineProject({
-		test: {
-			include: ["**/__tests__/**/*.{test,spec}.{ts,js,tsx,jsx}"],
-			setupFiles: "./vite.setup.ts",
+export default defineWorkersProject({
```

Comment:
Does this make the tests run in workerd, even if it's not a real Worker? If yes, is HTMLRewriter not just available as a global there? Do we even still need a polyfill?

Reply:
The polyfill is for the wrangler dev flow, which runs this in node

Reply:
Makes sense!

Reply:
Because this is Pages and not Workers+Assets?